src/server/pps/cmds/cmds.go

```diff
@@ -319,11 +319,8 @@ $ pachctl flush-job foo/XXX -p bar -p baz
 			if raw {
 				return marshaller.Marshal(os.Stdout, datumInfo)
 			}
-			writer := tabwriter.NewWriter(os.Stdout, 10, 1, 3, ' ', 0)
-			pretty.PrintDetailedDatumInfo(writer, datumInfo)
```

Comment:
@sjezewski I think you originally implemented this. It struck me as really weird because we're writing non-tab delimited data (via `PrintDetailedDatumInfo`) to a `tabwriter` which I think can lead to some pretty weird output. I tried it though and the output doesn't look that weird so I'm not sure what's going on. But I don't think this works generally, so just wanted to call it to your attention.

Reply:
Agreed, that is weird. Probably was a copy/paste error. Thanks